devtron-labs/dashboard
# #502 security module clair integration

# Description
To make CICD light we are breaking Security module (Clair) separately and listing it on the modules pages. So if user wants it he can install it.

Divide CICD into 2 part
CICD and Security module (Clair)

Fixes # [(issue)](https://github.com/devtron-labs/dashboard/issues/502)

## Type of change


# How Has This Been Tested?

- verified security option should not be there in primary navigation if Security module is not installed
- Go to Deploy image section and click on Show source info the Security tab should only be visible  if Security module is installed
- Go to CI Pipeline configuration the Scan for vulnerabilities box on the build page should only be visible  if Security module is installed
- Go to Build history page the Security tab should only be visible  if Security module is installed


# Checklist:
